examples/button_int.rs

```diff
@@ -31,6 +31,7 @@ fn EXTI0() {
 fn main() -> ! {
     let device_periphs = pac::Peripherals::take().unwrap();
     let mut reset_and_clock_control = device_periphs.RCC.constrain();
+    let syscfg = device_periphs.SYSCFG.constrain(&mut reset_and_clock_control.apb2);
```

Comment:
Note to self: look up the APB2 register

Reply:
I am not familiar with it, just  figured from the docs that I need SYSCFG configured, so I passed in whatever the HAL asked me to pass in.